src/main/java/core/basesyntax/Application.java

```diff
@@ -1,12 +1,11 @@
 package core.basesyntax;
 
 public class Application {
-    private static final int numOfBalls = 3;
+    private static final int BALLS_NUMBER = 3;
 
     public static void main(String[] args) {
-        Lottery randomBalls = new Lottery();
-        for (int i = 0; i < numOfBalls; i++) {
-            System.out.println(randomBalls.getRundomBall());
+        for (int i = 0; i < BALLS_NUMBER; i++) {
+            System.out.println(new Lottery().getRandomBall().toString());
```

Comment:
The same situation here. Can we create an instance of the Lottery class only once?